site/components/product/ProductCard/ProductCard.module.css

```diff
@@ -69,16 +69,11 @@
 }
 
 .imageContainer {
-  @apply flex items-center justify-center overflow-hidden;
-}
-
-.imageContainer > div {
```

Comment:
This was changed because there's no wrapper now, correct?

Reply:
yes, there was also a div wrapper removed from the component.